tests/performance_tests/performance_test_basic.py

```diff
@@ -97,9 +215,11 @@ def checkTotalTrx():
     testGenerationDurationSec = 60
     targetTps = 1
     transactionsSent = testGenerationDurationSec * targetTps
+    cont = chainsContainer()
 
-    waitForEmptyBlocks(node1)
-    setupTotal = checkTotalTrx()
+    # Get stats prior to transaction generation
+    cont.startBlock = waitForEmptyBlocks(node1)
+    fetchStats(cont.preData)
```

Comment:
It would be great if we could push all log processing to the end of the test as a post processing step.  This would mean we would need to process the whole log in one fell swoop, then be able to pull out a range of blocks that we want to gather stats on.